packages/app/src/cli/services/bulk-operations/format-bulk-operation-status.ts

```diff
@@ -0,0 +1,27 @@
+import {GetBulkOperationByIdQuery} from '../../api/graphql/bulk-operations/generated/get-bulk-operation-by-id.js'
+import {outputContent, outputToken, TokenizedString} from '@shopify/cli-kit/node/output'
+
+export function formatBulkOperationStatus(
+  operation: NonNullable<GetBulkOperationByIdQuery['bulkOperation']>,
+): TokenizedString {
+  switch (operation.status) {
+    case 'RUNNING':
+      return outputContent`Bulk operation in progress... ${outputToken.gray(
+        `(${String(operation.objectCount)} objects)`,
+      )}`
+    case 'CREATED':
+      return outputContent`Starting...`
+    case 'COMPLETED':
+      return outputContent`Bulk operation succeeded. ${outputToken.gray(`(${String(operation.objectCount)} objects)`)}`
```

Comment:
Instead of `Bulk operation succeeded. (12 objects)`, what about any of these?
- `Bulk operation succeeded: 12 objects`
- `Bulk operation succeeded (12 objects)`

Reply:
Great suggestion, I adjusted it to `Bulk operation succeeded: 12 objects` for now.

Reply:
TY!